examples/ads.amp.html

```diff
@@ -557,6 +558,17 @@ <h2 id="ezoic">Ezoic</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="felmat">felmat</h2>
+  <amp-ad width=300 height=250
+      type="felmat"
+      data-host="felmat.net"
+      data-fmt="banner"
+      data-fmk="U12473_n2cJD"
+      data-fmp="">
```

Comment:
Thank you for your review. 
I changed it to set the value of data-fmp to 0.
Please check it.